examples/passport-login/src/authentication-strategies/facebook.ts

```diff
@@ -0,0 +1,99 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: example-passport-oauth2-login
```

Comment:
Incorrect module name. The `lb4 copyright` command will fix it.

Reply:
agreed, fixed now